### ISSUE TITLE
Orbit Windows Install Not Working

### ISSUE THREAD

**Issue by gatordevin** [c-rus/orbit#4]
When running the windows installer I receive the following error. 
```
installation size: 10.74 MB
default installation path: C:\Users\gator\AppData\Local\Programs
enter installation path or press enter to continue:
set installation path: C:\Users\gator\AppData\Local\Programs
Install? [y/n]
y
error: The system cannot find the file specified. (os error 2)
press enter to exit
```

It successfully creates an orbit directory at the folder location specified but fails after that.

**To Reproduce**
1. Download release orbit-0.8.7-x86_64-windows.zip
2. Extract folder into Downloads
3. Run install file
4. Press enter to use default install path and y to install to see error

**Expected behavior**
I would expect the install to continue rather than the script to fail.

**Desktop (please complete the following information):**
 - OS: Windows 11 64 bit
 - Version 10.0.22621

**Comment by gatordevin** [comment]
Pretty sure the issue is here.
https://github.com/c-rus/orbit/blob/0e745d40b697afc2984ff46952b9d27010da5071/src/bin/install.rs#L100-L110

Particularly this line here.
https://github.com/c-rus/orbit/blob/0e745d40b697afc2984ff46952b9d27010da5071/src/bin/install.rs#L108

If I make a directory called orbit in the target install directory, it gets deleted, which indicates it's at least making it to this line, and it is failing to find a folder with the name it just copied over. This means it cannot rename it to orbit in the target install directory. 
I guess it shouldn't affect orbit working it just means the path is different, or I have to rename the folder to orbit manually.

**Comment by gatordevin** [comment]
Here is a pull request that should fix the issue #5 . It just replaces the relative reference for renaming the folder with an absolute reference.

**Comment by chaseruskin** [comment]
The issue is fixed with commit 16e909d6719f53c39eba3387f.
